sugondat/chain/pallets/length-fee-adjustment/src/lib.rs

```diff
@@ -139,6 +140,30 @@ pub mod pallet {
     pub type TargetBlockSize<T: Config> =
         StorageValue<_, Perquintill, ValueQuery, TargetBlockSizeDefault>;
 
+    /// Genesis config for setting up `NextLengthMultiplier` and `TargetBlockSize` storage.
+    /// Set None if the default values `NextLengthMultiplierDefault` & `TargetBlockSizeDefault` are to be used.
+    #[pallet::genesis_config]
+    #[derive(frame_support::DefaultNoBound)]
+    pub struct GenesisConfig<T: Config> {
+        /// Genesis set up for next length multiplier storage item
+        pub next_length_multiplier: Option<FixedU128>,
```

Comment:
`Multiplier` is an alias for `FixedU128`, it is used by `pallet_transaction_payment` and was also used in the file. Therefore, it should be used here as well